src/mrpro/data/KTrajectory.py

```diff
@@ -85,13 +89,33 @@ def from_tensor(
             This should be a 5-dim tensor, with (kz,ky,kx) stacked in this order along stack_dim
         stack_dim
             The dimension in the tensor the directions have been stacked along.
+        axes_order
+            Order of the axes in the tensor. Our convention usually is zyx order.
         repeat_detection_tolerance
             detects if broadcasting can be used, i.e. if dimensions are repeated.
             Set to None to disable.
         grid_detection_tolerance
             tolerance to detect if trajectory points are on integer grid positions
+        encoding_matrix
+            if an encoding matrix is supplied, the trajectory is rescaled to fit
+            within the matrix. Otherwise, it is left as-is.
```

Comment:
```suggestion
            If an encoding matrix is provided, the trajectory is normalized and 
            rescaled to fit within the dimensions of the matrix. If not provided, 
            the trajectory remains unchanged.

```

Reply:
and maybe add an example like
`i.e. SpatialDimension(z=1, y=64, x=64) for a single slice acquisition with 64x64 pixels`